guides/office/dress_code.md

```diff
@@ -0,0 +1,21 @@
+# Dress Code
+
+Made Tech is typically a casually dressed environment, however, we should keep in mind that we remain a professional organisation.
+
+This should be considered a rough guide as to acceptable dress in the Made Tech office. If you're working on-site or visiting a customer organisation you should be sensitive to their dress code and modify your dress accordingly. If in doubt please seek advice from another member of the team.
+
+## Generally OK for the office:
+
+- Jeans
+- T-shirts
+- Trainers
+- Flip flops
+- Smarter shorts
+- Hoodies
+
+
+## Generally not OK for the office:
+
+- Tracksuit bottoms / sweatpants
```

Comment:
I also feel vests/singlets are inappropriate, can we add this?

Reply:
Good catch!

Reply:
I'm probably going to regret opening this can of worms, but I feel like vest tops can be considered relatively smart for people identifying as female, whereas for those identifying as males much less so.